src/pages/CarForm.tsx

```diff
@@ -0,0 +1,300 @@
+
+  useEffect(() => {
+    if (dataInsertCar) {
+      Swal.fire({
+        icon: 'success',
+        title: 'Car created',
+        text: 'Car was created successfully',
+      }).then(() => navigate('/dashboard'));
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [dataInsertCar]);
+
+  const onSubmit = (data: IFormInputs) => {
+    const object = {
+      batch: testBatch ?? uuidv4(),
+      title: data.title,
+      brand_id: data.brand,
+      model_id: data.model,
+      color_id: data.color,
+      odometer: data.odometer,
+      sale_date: data.sale_date,
+      city_id: data.city,
+      state_id: data.state,
+      year: data.year,
+      price: data.price,
+      condition: data.condition,
+      vin: data.vin,
+    };
+
+    insertCarsOneMutation({
+      variables: {
+        object,
+      },
+      optimisticResponse: {
+        insert_cars_one: {
+          id: Number((Math.random() * 100).toFixed(0)),
+          ...object,
+        },
+      },
+      update(cache, { data }) {
+        cache.modify({
+          fields: {
+            cars: (existingFieldsData) => {
+              return [...existingFieldsData, data?.insert_cars_one];
+            },
+          },
+          optimistic: true,
+        });
+      },
+    });
+  };
+
+  if (fieldsError) {
+    return (
+      <ContainerCreateCar>
+        <Error type="normalError">There was an error</Error>
+      </ContainerCreateCar>
+    );
+  }
+
+  return (
+    <ContainerCreateCar>
+      {fieldsLoading ? (
+        <Spinner />
+      ) : (
+        <FormProvider {...methods}>
+          <ContainerForm>
+            <Title>Create Car</Title>
+            <FormCreateCar>
+              <div>
+                <label>Title</label>
+                <input
+                  type="text"
+                  {...register('title')}
+                  data-testid={'input-title'}
+                />
+                {errors?.title?.message && (
+                  <Error type="warningError">{errors?.title?.message}</Error>
+                )}
+              </div>
+
+              <BrandModelDropdowns brands={fieldsData?.brands} />
+
+              <Dropdown
+                label="Color"
+                fieldName="color"
+                options={fieldsData?.colors}
+              />
+
+              <div>
+                <label>Odometer</label>
+                <input
+                  type="number"
+                  {...register('odometer')}
+                  data-testid={'input-odometer'}
+                />
+                {errors?.odometer?.message && (
+                  <Error type="warningError">{errors?.odometer?.message}</Error>
+                )}
+              </div>
+
+              <div>
+                <label>Sale Date</label>
+                <input
+                  type="date"
+                  {...register('sale_date')}
+                  min={moment().format('YYYY-MM-DD')}
+                  data-testid={'input-sale-date'}
```

Comment:
if you hook up the input with the label you can avoid having to have a test id here to target these inputs

Reply:
I fixed it